src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -144,133 +144,218 @@ All parameters are case-insensitive.</value>
-    commands to PowerShell that require complex quotation marks or curly braces.
 
-    Example:
-        $command = 'dir "c:\program files" '
-        $bytes = [System.Text.Encoding]::Unicode.GetBytes($command)
-        $encodedCommand = [Convert]::ToBase64String($bytes)
-        pwsh -encodedcommand $encodedCommand
+    Accepts a base64-encoded string version of a command. Use this parameter to
+    submit commands to PowerShell that require complex quotation marks or curly
+    braces. The string must be formatted using UTF-16 character encoding.
 
--ExecutionPolicy | -ex | -ep
-    Sets the default execution policy for the current session and saves it
-    in the $env:PSExecutionPolicyPreference environment variable.
-    This parameter does not change the PowerShell execution policy
-    that is set in the registry.
+    For example:
 
-    Example: pwsh -ExecutionPolicy RemoteSigned
 
--File | -f
-    Default parameter if no parameters is present but any values is present in the command line.
-    Runs the specified script in the local scope ("dot-sourced"), so that the functions
-    and variables that the script creates are available in the current session.
-    Enter the script file path and any parameters. File must be the last parameter
-    in the command, because all characters typed after the File parameter name are interpreted
-    as the script file path followed by the script parameters.
+    $command = 'dir "c:\program files" '
+    $bytes = [System.Text.Encoding]::Unicode.GetBytes($command)
+    $encodedCommand = [Convert]::ToBase64String($bytes)
+    pwsh -encodedcommand $encodedCommand
 
-    Example: pwsh HelloWorld.ps1
+-ExecutionPolicy | -ex | -ep
 
--Help | -h | -? | /?
-    Shows this help message.
+    Sets the default execution policy for the current session and saves it in
+    the "$env:PSExecutionPolicyPreference" environment variable. This parameter
+    does not change the PowerShell execution policy that is set in the
+    registry.
 
 -InputFormat | -in | -if
-    Describes the format of data sent to PowerShell.
-    Valid values are "Text" (text strings) or "XML" (serialized CLIXML format).
+
+    Describes the format of data sent to PowerShell. Valid values are "Text"
+    (text strings) or "XML" (serialized CLIXML format).
 
 -Interactive | -i
-    Present an interactive prompt to the user. Inverse for NonInteractive parameter.
+
+    Present an interactive prompt to the user. Inverse for NonInteractive
+    parameter.
 
 -LoadProfile | -l
-    Load the PowerShell profiles. This is the default behavior even if this is not specified.
+
+    Load the PowerShell profiles. This is the default behavior even if this is
+    not specified.
 
 -NoExit | -noe
+
     Does not exit after running startup commands.
 
-    Example: pwsh -NoExit -Command Get-Date
+    Example: "pwsh -NoExit -Command Get-Date"
 
 -NoLogo | -nol
-    Hides the copyright banner at startup.
+
+Hides the copyright banner at startup.
```

Comment:
Filed <https://github.com/PowerShell/PowerShell/issues/13516> about the removal of indentation.